caas/kubernetes/provider/exec/errors.go

```diff
@@ -54,3 +56,41 @@ func handleContainerNotFoundError(err error) error {
 	}
 	return containerNotRunningError(match[1])
 }
+
+type execRetriableError struct {
```

Comment:
Retryable is the more correct spelling. Both are correct but in the context of software Retryable is more correct. We already use "Retryable" in juju not Retriable.

Reply:
no worries, change it to `Retryable` lol